src/rust/src/wallet.rs

```diff
@@ -686,19 +688,23 @@ impl Wallet {
             .collect()
     }
 
-    pub fn note_commitment_tree_root(&self) -> MerkleHashOrchard {
-        self.witness_tree.root()
+    /// Returns the note of the Orchard note commitment tree, as of the specified checkpoint
```

Comment:
```suggestion
    /// Returns the root of the Orchard note commitment tree, as of the specified checkpoint
```

Reply:
Fixed in https://github.com/zcash/zcash/pull/5934